cdn/snippets.py

```diff
@@ -27,6 +27,7 @@
 import argparse
 import base64
 from datetime import datetime
```

Comment:
nit: could you put this on one line 
```
from datetime import datetime, timezone 
```

Reply:
Resolved in f3b87f0ef11defcb1efdd4b8d140981fa00406db of the new pull request